src/css/variables.module.scss

```diff
@@ -7,7 +7,9 @@
 }
 
 $theme-filter: "invert(93%) hue-rotate(180deg)";
+$right-sidebar-width: "302px";
```

Comment:
`docked-sidebar-width` for consistency ?

Reply:
the width is both for docked and floating sidebar

Reply:
true ok let it be